src/c4/C4_Serial.cc

```diff
@@ -116,6 +116,12 @@ void wait_all(unsigned /*count*/, C4_Req * /*requests*/) {
   return;
 }
 
+std::vector<int> wait_all_with_source(unsigned /*count*/,
+                                      C4_Req * /*requests*/) {
+  // Insist(false, "no messages expected in serial programs!");
```

Comment:
Why no Insist? 

On the other hand, if the function could be called and pass silently, does the return value need to be of the same length as `requests`, and do the ints need to have a value that couldn't be silently interpreted as "hey that worked!".

Reply:
@timmah I think we want to serial case to pass as a a no-op call.  This is what we do in other parts of c4.

Reply:
OK

Reply:
Honestly, I just copy-pasted from the other wait_all. I mean to ask about the commented-out Insist to see if there was any reason to restore it... from KT's comment, I'm guessing no. I'll remove it!